documentation/notes/joss_paper/paper.md

```diff
@@ -47,11 +47,10 @@ A future development is planned that will enable the user to specify the basis o
 
 The model implementation consists of submodules to set up the model's parameters and to compute the tensor that defines the coefficients in the tendencies of the model variables; more details can be found in @DDV2016.
 This tensor is used by the code to compute the tendencies function and its Jacobian matrix. These functions can then be fed to the `qgs` built-in Runge-Kutta integrator or 
-to another integrator implemented by the user. As an example, the usage of the Julia DifferentialEquations.jl [@RN2017] integration package through the Python diffeqpy [@diffeqpy] package is provided.
-Technical details about this implementation can be found in the *Code Description* section of the included documentation.
```

Comment:
I suggest to remove this sentence ("Technical details...") because I think it can be taken as given that further details, including about implementation, can be found in the documentation, though this is merely a suggestion and if you wish to keep it despite this comment, that is fine and won't influence the review outcome. You could consider putting it as a footnote, which would be a way to retain it without having it interrupt the flow of the text.

Reply:
I think I will retain it as a footnote.